### ISSUE TITLE
Need to check shiny when there's no international and no charm

### ISSUE THREAD

**Issue by hirakujira** [Quandra/SMHatchingRNGTool#12]
https://github.com/Quandra/SMHatchingRNGTool/blob/master/SMHatchingRNGTool/EggRNGSearch.cs#L143

I think you may need to check PSV and TSV before this line? 
In your code, if there's no international hatching and no charm, it's not possible to hatch a shiny egg. 

I guess that the "暗号化定数 -- Encryption Constant" is not encryption constant, it's for checking whether the egg is shiny or not in the first time.

I'm not professional, so I'm not sure whether I'm wrong. But the shiny egg should happen even when you have no international and no charm. :)

**Comment by shadowofdarkness** [comment]
The PID is generated differently if you are not using Masuda and/or Charm. That is why at least one is required.

**Comment by hirakujira** [comment]
I meant, you didn't check shiny if you are "not" using Masuda and/or Charm.  But in this cause, it still have shiny (1/4096)

**Comment by shadowofdarkness** [comment]
Yes you can still get shiny Pokemon without either Masuda or Charm. This program is just unable to predict it due to the PID being generated from a different source and not derived from the same seed.
